src/shared/components/FlowPlayer/FlowPlayer.scss

```diff
@@ -0,0 +1,86 @@
+.flowplayer {
+	width: 100%;
+	height: 100%;
+	vertical-align: bottom;
+
+	.fp-title {
+		text-transform: none;
+	}
+
+	.is-fullscreen .fp-footer {
+		display: inline-block;
+	}
+
+	.fp-footer {
+		display: none;
+		max-width: calc(100% - 3.2rem);
+		top: 1em;
```

Comment:
Waarom `em` gebruiken?

Reply:
Mono-style copy paste. Heb het gefixt het in de volgende skinning PR. :)